### PR TITLE
fix(apm infrastructure): adding information about APM and infrastructure

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 1]
```diff
@@ -15,22 +15,76 @@ redirects:
...
+
+import apmCPUPercentageViolationinAPM from 'images/apm_screenshot-crop_CPU-percentage-violation-in-APM.webp'
+
+import apmToggleBetweenHistogramViews from 'images/apm_screenshot-crop_toggle-between-histogram-views.webp'
+
+import apmExploringaSpecificHostinAPM from 'images/apm_screenshot-full_exploring-a-specific-host-in-APM.gif'
+
+import apmFilterbyApplication from 'images/apm_screenshot-full_filter-by-application.webp'
 
 The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data lets you see your APM data and infrastructure data side by side so you can find the root cause of problems more quickly.
```
Comment (akristen) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:28]: This intro can be a little bit neater. I would look at M's original message and maybe string a narrative about correlating app performance with infra health. Root cause is one part of this, but I think we can get more detailed with pointing out the 'app deployment --> affects resources' vs 'infra resources --> affects app performance'

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 2]
```diff
@@ -15,22 +15,76 @@ redirects:
...
-The main ways to find and use APM data in infrastructure monitoring are:
+First, this doc will walk you through the process of resolving infrastructure issues with APM. Then it will dig deeper into some of the key features of APM and infrastructure monitoring. 
 
-* [View APM charts on infrastructure monitoring UI pages](#apm-chart)
-* [Filter hosts by application data](#filter-by-app)
-* [Switch between infrastructure and APM](#switch)
-* [Examine APM data in Inventory and Events pages](#inventory-events)
+<Steps>
+    <Step>
+    ## Do you have a problem in your system?
```
Comment (akristen) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:34]: I'd err on the side of describing the `thing` to be solved for in the title, so users know what action theyre going to take. It looks like the action here is 'respond to an alert', so maybe just making that (or something like that) the title

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 3]
```diff
@@ -15,22 +15,76 @@ redirects:
...
-* [View APM charts on infrastructure monitoring UI pages](#apm-chart)
-* [Filter hosts by application data](#filter-by-app)
-* [Switch between infrastructure and APM](#switch)
-* [Examine APM data in Inventory and Events pages](#inventory-events)
+<Steps>
+    <Step>
+    ## Do you have a problem in your system? 
 
-Infrastructure data appears in <InlinePopover type="apm" /> in the host table on the [APM Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page.
+    You're the engineer responsible for the `Billing Service` application and you get an alert that says, "Error percentage > 45% for at least five minutes on `Billing Service`." The first thing you're going to do is go to the `Billing Service` application in APM and open the **Summary** page to get an overview of the health of your system. The first indicator of a problem in your system is a high Apdex score, which is a measure user satisfaction. Here you can see that the score is .79 and has triggered a critical violation. Next you're going to check your error rate. Here you can see that the error rate has hit 100%.
```
Comment (akristen) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:36]: I'd break this up, maybe using bulleted steps to separate the actions that an eng/dev might take here. I also think you could add the preamble/context situation *before* the steps, making it clear that the stepped tutorial is walking User through an example use case

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 4]
```diff
@@ -15,22 +15,76 @@ redirects:
...
-You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
+
+  <img
+    title="Apdex violations"
+    alt="A screenshot depicting critical violations in apdex"
+    src={apmCriticalViolationsinApdex}
+    />
+    </Step>
+    <Step>
+   ## Where does the problem lie?
```
Comment (akristen) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:48]: Possible stand in title: Diagnose root cause

(or some other verb besides 'diagnose'. I like 'diagnose' because it implies you're trying to whittle down multiple causes)

Comment (akristen) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:48]: actually maybe not 'diagnose root cause', but something more specific like 'Determine the source of the error' or some equivalent.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 5]
```diff
@@ -15,22 +15,76 @@ redirects:
...
+
+  <figcaption>
+    Inspect your host without leaving the APM summary page. 
+  </figcaption>
+  </Step>
+</Steps>
+
+Now that you know how to troubleshoot with APM and infrastructure monitoring we're going to explore how to integrate APM and infrastructure data and put it into practice. 
 
 ## How to integrate APM and infrastructure data [#how-it-works]
```
Comment (akristen) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:89]: I would put this before the stepped tutorial IMO

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 6]
```diff
@@ -42,6 +96,20 @@ For <InlinePopover type="apm" /> and infrastructure data to be integrated, all o
 
 If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
 
+## View logs for your APM and infrastructure data [#logs-context]
+
+You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
```
Comment (akristen) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:101]: "no need to switch to another UI page" is one of the benefits of using logs in context, and i think the value prop for the whole doc is being able to synthesize as many data sources as possible without leaving a single UI page

**Review by akristen — COMMENTED** [review]
I'd look into some structural changes re: breaking up steps, and maybe get a SME review to make the example tutorial a little meatier + maybe get a strong value prop down too (for the story)

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:34]: great! thanks :)

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 3 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:36]: sounds good! Thanks.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:48]: done!

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 5 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:89]: great idea

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 7]
```diff
@@ -15,70 +15,104 @@ redirects:
...
 
-* The APM agent and the infrastructure agent must be installed on the same host.
-* Both agents must use the same <InlinePopover type="licenseKey" />.
-* They must use the [same hostname](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure#hostnames).
+    * The APM agent and the infrastructure agent must be installed on the same host.
+    * Both agents must use the same <InlinePopover type="licenseKey" />.
+    * They must use the [same hostname](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure#hostnames).
 
-If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:42]:
```suggestion
If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
```

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: APM data in infrastructure monitoring
+title: Infrastructure monitoring within APM
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:2]: I think 'within' is a bit awkward, might just be me. Maybe 'Infrastructure data on APM Summary page'? 
Also ensure you align the nav doc title with this title.

**Review by Mesverrum — COMMENTED** [review]
LGTM from an SME perspective

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 7 above)
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx]:
```suggestion
    If the integration is not working, see [Troubleshooting the APM-infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
```

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx]: Fixed this!

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 9]
```diff
@@ -15,70 +15,104 @@ redirects:
...
-If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    </Step>
+    <Step>
+    ## Respond to an alert
 
-## View APM charts [#apm-chart]
+    In this example, let's say that you're the engineer responsible for the `Billing Service` application and you get an alert that says, "Error percentage > 45% for at least five minutes on `Billing Service`." 
+    
+    * The first thing you're going to do is go to the `Billing Service` application in APM and open the **Summary** page to get an overview of the health of your system.  A high Apdex score, which is a measure user satisfaction, can indicate that there's a problem in your system. Here you can see that the score is .79 and has triggered a critical violation.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:49]:
```suggestion
    * The first thing you're going to do is go to the `Billing Service` application in APM and open the **Summary** page to get an overview of the health of your system.  A high Apdex score, which is a measure of user satisfaction, can indicate that there's a problem in your system. Here you can see that the score is .79 and has triggered a critical violation. 
```

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 10]
```diff
@@ -15,70 +15,104 @@ redirects:
...
+    <img
+    title="CPU violation in APM"
+    alt="A screenshot depicting a CPU violation that corresponds to a deployment marker."
+    src={apmCPUPercentageViolationinAPM}
+    />
+  </Step>
+    <Step>
+     ## Dig deep into a specific host
+      
+      Now that you know that a recent deployment in your `Billing Service` application caused a spike in errors and critical Apdex violations you might want to look into a specific host for more clarity. Click the name of the host you want to inspect. It will reveal a sidebar that imports all relevant information from the **Infrastructure** page. This allows you to access all the information you need regarding your host and any service errors without leaving the rest of your data.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:87]: maybe 'without leaving your place in the UI'

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 11]
```diff
@@ -15,70 +15,104 @@ redirects:
...
-You can switch from APM to infrastructure from the host table on the APM [Summary](#infra-apm-overview) page.
+When your <InlinePopover type="apm" /> and infrastructure data is linked, you can filter displayed host data by searching for the specific application you want to inspect. In the case above, you would want to filter for `Billing Service`.
+
+<img
+    title="filter by application"
+    alt="A screenshot showing you how to filter the infrastructure page by application"
+    src={apmFilterbyApplication}
+/>
 
 ## APM data in Inventory and Events [#inventory-events]
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:117]: maybe 'APM data on inventory and events UI pages'. Don't think we can capitalize these (because we can't use bold in headers, and think about how that would look for UI pages with two or more words in name).

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 12]
```diff
@@ -88,7 +122,7 @@ When your <InlinePopover type="apm" /> and infrastructure data is linked, you ca
 
 When your <InlinePopover type="apm" /> and infrastructure data is linked, you have more available host data in APM.
 
-The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page contains a table with data about your app's hosts and instances, including:
+The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page contains a **Infrastructure** section with data about your app's hosts and instances, including:
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:125]: i may be misundesrtanding, but this seems a bit redundant at this point: haven't we already discussed this? also, more generally, I think it's overdocumenting to simply include a list of what's in the UI: this is redundant (we could just say 'provides data about' and give a general description. Also creates maintenance issues (if the exist list of metrics changes). So we avoid unnecessary overdocumentation and futureproof the docs more when we're more vague and just give a general description versus exact description.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 13]
```diff
@@ -97,9 +131,7 @@ The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page
 * CPU usage
 * Memory
 
-You can toggle between a table view or breakout metric details for the individual hosts by selecting **View table** or **Break out each metric by host**.
-
-For more information on host data on the APM **Summary** page, see [host details](/docs/apm/applications-menu/monitoring/apm-overview-page#host_details).
+To investigate a specific host, click its name and it will reveal an expanded view on the right-hand side of the page.
 
 ## Troubleshoot missing APM data [#troubleshooting]
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:136]: I think with what you have at top with requirements, you can delete this section now, as it's already covered.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 7 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx]: done

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx (hunk 12 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:125]: agree!

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 14]
```diff
@@ -15,92 +15,111 @@ redirects:
...
 
-## View logs for your APM and infrastructure data [#logs-context]
-
-You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
-
-## How to integrate APM and infrastructure data [#how-it-works]
+The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data allows you to see the health of your entire system from a single page. On the APM **Summary** page you can monitor hosts, apps, events, and alerts activity and use embedded change tracking to compare your data with any recent deployments. From one page you can respond to an alert, identify a root cause, and quickly resolve any impacts to host performance.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:28]: Overall this is really good value prop and a strong hook. For the final sentence, I would put the "From one page" part at the end, but overall I like this intro a lot

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 15]
```diff
@@ -15,92 +15,111 @@ redirects:
...
-
-## How to integrate APM and infrastructure data [#how-it-works]
+The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data allows you to see the health of your entire system from a single page. On the APM **Summary** page you can monitor hosts, apps, events, and alerts activity and use embedded change tracking to compare your data with any recent deployments. From one page you can respond to an alert, identify a root cause, and quickly resolve any impacts to host performance. 
 
-For <InlinePopover type="apm" /> and infrastructure data to be integrated, all of the following must be true:
+First, this doc will walk you through the process of resolving infrastructure issues with APM. Then it will dig deeper into some of the key features of APM and infrastructure monitoring.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:30]: I think there should be a comma after "Then" since this is a procedural string, but I've been experimenting a lot with "what this doc is going to tell you" lead-ins and they're always good to have so it's good that this is here.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 16]
```diff
@@ -15,92 +15,111 @@ redirects:
...
-* They must use the [same hostname](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure#hostnames).
+<Steps>
+  <Step>
+    ## Integrate APM and infrastructure data [#how-it-works]
 
-If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    For <InlinePopover type="apm" /> and infrastructure data to be integrated, all of the following must be true:
 
-## View APM charts [#apm-chart]
+    * The APM agent and the infrastructure agent must be installed on the same host.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:38]: I think we're taking periods out of bulleted lists like this now. Awesome use of pop-overs though.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 17]
```diff
@@ -15,92 +15,111 @@ redirects:
...
-When your <InlinePopover type="apm" /> and infrastructure data is linked, you have access to APM data charts on these Infrastructure monitoring UI pages: **Hosts**, **Network**, **Storage**, and **Processes**.
+If the integration is not working, see [Troubleshooting the APM-infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    </Step>
+    <Step>
+    ## Respond to an alert
 
-To switch to different charts: select the dropdown beside a chart's name and choose a new chart. Application-related charts will be near the top.
+    In this example, let's say that you're the engineer responsible for the `Billing Service` application and you get an alert that says, "Error percentage > 45% for at least five minutes on `Billing Service`." 
+    
+    * The first thing you're going to do is go to the `Billing Service` application in APM and open the **Summary** page to get an overview of the health of your system.  A high Apdex score, which is a measure of user satisfaction, can indicate that there's a problem in your system. Here you can see that the score is .79 and has triggered a critical violation.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:49]: Probably can shorten this to just "First, go to the `Billing Service`..."

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 18]
```diff
@@ -15,92 +15,111 @@ redirects:
...
+    alt="A screenshot depicting critical violations in apdex"
+    src={apmCriticalViolationsinApdex}
+    />
+    </Step>
+    <Step>
+   ## Determine the source of your errors
 
-* From the [host filter](/docs/infrastructure/new-relic-infrastructure/filter-group/filter-sets-organize-your-infrastructure-hosts), select **Applications**.
-* Select the application you want to filter on.
+    Scroll down to the **Infrastructure** section of the APM summary page. Here you'll see a table that lists each host connected to the `Billing Service` application and a record of their **Response time**, **Throughput**, **Error rate**, **CPU%**. and **Memory %**. Below the chart are histograms that highlight two of these golden signals. The default selections are **CPU %** and **Memory %** but you can also click the dropdown menu in the top left and select a different view.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:64]: I think there should be a comma after "Here"

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 19]
```diff
@@ -15,92 +15,111 @@ redirects:
...
+    <img
+    title="CPU violation in APM"
+    alt="A screenshot depicting a CPU violation that corresponds to a deployment marker."
+    src={apmCPUPercentageViolationinAPM}
+    />
+  </Step>
+    <Step>
+     ## Dig deep into a specific host
+      
+      Now that you know that a recent deployment in your `Billing Service` application caused a spike in errors and critical Apdex violations you might want to look into a specific host for more clarity. Click the name of the host you want to inspect. It will reveal a sidebar that imports all relevant information from the **Infrastructure** page. This allows you to access all the information you need regarding your host and any service errors without leaving the rest of your data.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:87]: I think there should be a comma after "Apdex violations"

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 20]
```diff
@@ -15,92 +15,111 @@ redirects:
...
 
-You can switch from APM to infrastructure from the host table on the APM [Summary](#infra-apm-overview) page.
+  <figcaption>
+    Inspect your host without leaving the APM summary page. 
+  </figcaption>
+  </Step>
+</Steps>
 
-## APM data in Inventory and Events [#inventory-events]
+Now that you know how to troubleshoot with APM and infrastructure monitoring we're going to explore how to integrate APM and infrastructure data and put it into practice.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:101]: I think there should be a comma after "infrastructure monitoring"

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 21]
```diff
@@ -15,92 +15,111 @@ redirects:
...
 
 ## Troubleshoot missing APM data [#troubleshooting]
 
-APM/Infrastructure integration should happen automatically if you have both the <InlinePopover type="apm" /> agent and the infrastructure agent installed on the same host(s) and they use the same <InlinePopover type="licenseKey" /> and have the same hostname set. If you do not see APM data in infrastructure monitoring, see [Troubleshooting](/docs/infrastructure/new-relic-infrastructure/troubleshooting/troubleshooting-apm-data-not-infrastructure).
+APM/Infrastructure integration should happen automatically if you have both the <InlinePopover type="apm" /> agent and the infrastructure agent installed on the same host(s) and they use the same <InlinePopover type="licenseKey" /> and have the same hostname set.
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:123]: The slash feels a bit awkward here, maybe just APM and Infrastructure?

Comment (jeff-colucci) [reply src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:123]: There are 2 ands at the end of this section, I think it should be "agent installed on the same hosts, they use the same license key, and have the same hostname set.

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 22]
```diff
@@ -15,92 +15,111 @@ redirects:
...
 
 ## Troubleshoot missing APM data [#troubleshooting]
 
-APM/Infrastructure integration should happen automatically if you have both the <InlinePopover type="apm" /> agent and the infrastructure agent installed on the same host(s) and they use the same <InlinePopover type="licenseKey" /> and have the same hostname set. If you do not see APM data in infrastructure monitoring, see [Troubleshooting](/docs/infrastructure/new-relic-infrastructure/troubleshooting/troubleshooting-apm-data-not-infrastructure).
+APM/Infrastructure integration should happen automatically if you have both the <InlinePopover type="apm" /> agent and the infrastructure agent installed on the same host(s) and they use the same <InlinePopover type="licenseKey" /> and have the same hostname set. 
+
+If you do not see APM data in infrastructure monitoring, see [Troubleshooting](/docs/infrastructure/new-relic-infrastructure/troubleshooting/troubleshooting-apm-data-not-infrastructure).
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:125]: do not to don't

File: src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx [hunk 23]
```diff
@@ -15,92 +15,111 @@ redirects:
...
-If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    For <InlinePopover type="apm" /> and infrastructure data to be integrated, all of the following must be true:
 
-## View APM charts [#apm-chart]
+    * The APM agent and the infrastructure agent must be installed on the same host.
+    * Both agents must use the same <InlinePopover type="licenseKey" />.
+    * They must use the [same hostname](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure#hostnames).
 
-When your <InlinePopover type="apm" /> and infrastructure data is linked, you have access to APM data charts on these Infrastructure monitoring UI pages: **Hosts**, **Network**, **Storage**, and **Processes**.
+If the integration is not working, see [Troubleshooting the APM-infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
```
Comment (jeff-colucci) [inline src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx:42]: "If the integration" should probably be "if your integration", helps establish the link between the reader and the doc. I almost didn't notice this, the rest of the doc does a great job of doing that!
